src/utils/uppy.js

```diff
@@ -0,0 +1,103 @@
+import Uppy from '@uppy/core';
+import XHRUpload from '@uppy/xhr-upload';
+import { MAX_FILE_SIZE, MAX_NUM_FILES } from '../config/settings';
+import { FILE_UPLOAD_ENDPOINT } from '../config/api';
+import { showErrorToast, showWarningToast } from './toasts';
+import { POST_FILE } from '../types';
+import { postMessage } from '../actions/common';
+import { FILE } from '../config/appInstanceResourceTypes';
+
+const configUppy = ({
```

Comment:
Let's change this to `configureUppy`, as `config` is usually used as an abbreviation for `configuration`, not the verb "to configure".